src/libraries/System.Text.Json/Common/JsonSimpleNamingPolicy.cs

```diff
@@ -0,0 +1,147 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Buffers;
+using System.Globalization;
+
+namespace System.Text.Json
+{
+    internal sealed class JsonSimpleNamingPolicy : JsonNamingPolicy
```

Comment:
You might also want to include a `DebuggerDisplayAttribute` annotation so that users know what type of naming policy they are inspecting. Alternatively, you could unseal the class and use implementation inheritance:
```C#
internal sealed class JsonSnakeCaseLowerNamingPolicy : JsonSeparatorNamingPolicy
{
     public JsonCamelCaseLowerNamingPolicy()
          : base('_', false)
     { }
}
```